src/main/java/org/isf/utils/treetable/AbstractTreeTableModel.java

```diff
@@ -60,21 +60,17 @@
  * intended for use in the design, construction, operation or
  * maintenance of any nuclear facility.
  */
-
 import javax.swing.event.EventListenerList;
 import javax.swing.event.TreeModelEvent;
 import javax.swing.event.TreeModelListener;
 import javax.swing.tree.TreePath;
  
 /**
  * An abstract implementation of the TreeTableModel interface, handling 
- * the list of listeners. 
- *
- * @version %I% %G%
+ * the list of listeners.
```

Comment:
Maybe it could be in one line only?

Reply:
Removed all the version %x% strings in the files copied from Sun Microsystems.